.nvmrc

```
@@ -0,0 +1 @@
+v20.9.0
```

How about we specify the major version only to match test.yml workflow?

```suggestion
v20
```

---

Let me test that locally to see if it works as intended. One sec

---

Yep, that works!

---

Even better if we centralised this and read the node version from .nvmrc file in the github action: https://github.com/actions/setup-node/issues/32#issuecomment-1003854758

---

@ndv99  This is more relevant for us: https://github.com/actions/setup-node/issues/32#issuecomment-872375574